pkgs/development/python-modules/fasm/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "fasm";
+  version = "0.0.2.post100";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
```

Comment:
According to https://pypi.org/project/fasm/0.0.2.post100/#files it doesn't have a source distribution on PyPI. `fetchFromGitHub` should be used instead of `fetchPypi`